### ISSUE TITLE
Facebook Store Location Pages to Match Brickwork

### ISSUE THREAD

**Issue by c21hornak** [Century21Stores/SPECIAL#69]
From the Manage Locations page in Facebook: https://www.facebook.com/century21stores/settings/?tab=locations 

Please build whatever pages are missing and take whatever steps are required to get them verified like our other Store Location Pages in Facebook. Refer to Brickwork for the full list of stores.

Once they are all created, please update the pertinent fields (address, phone, store hours, etc.) so they match Brickwork.

**Comment by juanmriosv** [comment]
Hi @c21hornak, 

I assigned this to @lstaab Lydia, but from what I understood yesterday. You are doing it?

**Comment by c21hornak** [comment]
@juanmriosv correct, I'll make sure all locations exist in Facebook first, then I can assign back to Lydia

**Comment by c21hornak** [comment]
@lstaab all locations exist now in Facebook - ok to begin updating with correct store information from Brickwork.

cc @juanmriosv

**Comment by lstaab** [comment]
@c21hornak -Ok thanks-- will work on today/tomorrow. Please note: As of 5/5 I updated all hrs but the 3 that were missing: Citipoint, C21 Edition & Sawgrass . So will finish hrs on those and then will proceed on to match other info from brickwork to FB on all. cc @juanmriosv

**Comment by lstaab** [comment]
@c21hornak cc @juanmriosv Hours done for all Facebook stores. Want to clarify what additional fields to start next to fill in on Facebook. Drafted a spreadsheet for About Page/Fields. Started with Store 1. Appreciate your feedback-some notes/questions in yellow. Thanks.
https://docs.google.com/spreadsheets/d/1_yYGHoInzMahnyl7CZWU3JR8OqvheIwBdTAPO1aJFU8/edit?usp=sharing

**Comment by c21hornak** [comment]
@lstaab I added my comments on your Google Doc.
